corehq/apps/geospatial/static/geospatial/js/gps_capture.js

```diff
@@ -182,26 +185,55 @@ hqDefine("geospatial/js/gps_capture",[
         self.saveDataRow = function (dataItem) {
             self.isSubmissionSuccess(false);
             self.hasSubmissionError(false);
+            let dataItemJson = ko.mapping.toJS(dataItem);
+            if (self.createCaseEnabled()) {
+                const caseType = $('#report_filter_case_type').val();
```

Comment:
Does the filter support choosing multiple case types?

Reply:
No, the user can only ever select one case type at a time.